### PR TITLE
FORMAT: resolve pep8 violations

### PR DESCRIPTION
**Pull request by ironmig**
Resolved pep8 violations in the python files. I intentionally excluded navigator_missions, because my future PR using mil_tasks will change almost every file in that package, so the merge conflicts would  be annoying. Most changes were done with the behavior-safe autopep8 and autoflake commands, so the reviewers need only look at the 3rd+ commits for verification. Thanks @yikaimao for working on this as well, it was a valuable reference in fixing the formating.

### CODE REVIEW

File: perception/navigator_vision/nodes/database_color.py [hunk 1]
```diff
@@ -114,58 +118,59 @@ def publish(self, *args):
...
         self.values.extend(values)
         self.dists.extend(dists)
         self.q_errs.extend(q_diffs)
 
-    def compute_confidence(self, (value_w, dist_w, q_diff_w), get_conf=False, **kwargs):
+    def compute_confidence(self, xxx_todo_changeme2, get_conf=False, **kwargs):
         """Don't try to compute weights with bad data (too few samples)"""
-
-        # Get some value information
+        (value_w, dist_w, q_diff_w) = xxx_todo_changeme2
```
Comment (RustyBamboo) [inline perception/navigator_vision/nodes/database_color.py]: :smile:

File: perception/navigator_vision/nodes/database_color.py [hunk 2]
```diff
@@ -306,70 +314,70 @@ def _get_closest_color(self, hue_angle):
...
-        
-        msg = t_color.as_message 
+
+        msg = t_color.as_message
         msg.id = totem_id
         msg.confidence = w
         msg.labels = ["value_errs", "dists", "q_diffs"]
         msg.weights = weights
-        msg.color = colors[0]
+        msg.color = color[0]
```
Comment (RustyBamboo) [inline perception/navigator_vision/nodes/database_color.py]: nice

Comment (ironmig) [reply perception/navigator_vision/nodes/database_color.py]: This script has so many bugs I have no idea why it's in the repo lol

File: perception/navigator_vision/nodes/database_color.py (hunk 1 above)
Comment (ironmig) [reply perception/navigator_vision/nodes/database_color.py]: So apparently autopep8 actually does this. Fixed

**Comment by RustyBamboo** [comment]
Not seeing anything else that could cause stuff to break. Guess we'll find errors as we go on...

**Comment by ironmig** [comment]
Ran in SIM, able to move and didn't see any errors. The manual changes were pretty minimal so I think we'll merge it fingers crossed.

**Pull request merged**